### PR TITLE
Rewrite `Game mode`

### PR DESCRIPTION
**Pull request by TicClick**
- fixes some sentences being unnecessarily blown out, and quirks like `Made to mimic the feel of playing a real drum, the player makes use of ...`

### CODE REVIEW

**Review by Joehuu — COMMENTED** [review]
Don't know about the mascot descriptions, but goes to show that mascots also needs a rewrite.

**Comment by TicClick** [comment]
thanks, wasn't sure how to describe the colours. as for the rewrite, yes, it's been probably pieced together as a stub and no one has really bothered to lay out it all together

File: wiki/Game_mode/en.md [hunk 1]
```diff
@@ -1,42 +1,50 @@
...
+osu! has four **game modes** with different mechanics, originally inspired by other rhythm games:
 
-Each game mode has its own icon/symbol: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] (osu!, osu!taiko, osu!catch, and osu!mania respectively).
+- ![][osu!] [osu!](#-osu!)
+- ![][osu!taiko] [osu!taiko](#-osu!taiko)
+- ![][osu!catch] [osu!catch](#-osu!catch)
+- ![][osu!mania] [osu!mania](#-osu!mania)
 
-The default game mode is osu!. However, upon start-up, osu! will open with the game mode that was last selected. To switch between game modes, the player can either click on the `Mode` button at the bottom of the Song Selection screen, or by pressing `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, or `Ctrl`+`4` to switch between osu!, osu!taiko, osu!catch, and osu!mania respectively.
+To switch between modes, click the `Mode` button at the bottom of the Song Selection screen, or use key combinations:
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:10]:
```suggestion
To switch between modes, click the `Mode` button at the bottom of the Song Selection screen, or use these key combinations:
```

File: wiki/Game_mode/en.md [hunk 2]
```diff
@@ -1,42 +1,50 @@
...
+
+- `Ctrl`+`1` for osu!
+- `Ctrl`+`2` for osu!taiko
+- `Ctrl`+`3` for osu!catch
+- `Ctrl`+`4` for osu!mania
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:15]:
```suggestion
- `Ctrl` + `1` for osu!
- `Ctrl` + `2` for osu!taiko
- `Ctrl` + `3` for osu!catch
- `Ctrl` + `4` for osu!mania
```

File: wiki/Game_mode/en.md [hunk 3]
```diff
@@ -1,42 +1,50 @@
...
 
 ![Mode selection interface](/wiki/shared/Interface_mode.png "Mode selection interface")
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:17]: this image could use an update

File: wiki/Game_mode/en.md [hunk 4]
```diff
@@ -1,42 +1,50 @@
...
 
 ## ![][osu!] osu!
 
 *Main page: [osu!](osu!)*
 
-*osu!* is the default game mode in the osu! client, and is based on the original [iNiS'](https://en.wikipedia.org/wiki/INiS "Wikipedia") rhythm video game *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. In osu!, the player is expected to tap hit circles, hold sliders, and spin spinners to the beat of songs.
+*osu!* is the default mode, where players are expected to tap hit circles, hold sliders, and revolve spinners to the beat of songs. It is based on the original [iNiS'](https://en.wikipedia.org/wiki/INiS "Wikipedia") rhythm video game *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. osu!'s mascot is [pippi](/wiki/Mascots#-pippi), a pink/black-haired girl.
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:23]: more cheesiness please

```suggestion
*osu!* is the default mode, where players are expected to tap circles, hold sliders, and spin spinners to the beat of the music. It is based on the original [iNiS](https://en.wikipedia.org/wiki/INiS "Wikipedia")' rhythm game *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. osu! has its own mascot called [pippi](/wiki/Mascots#-pippi), a pink/black-haired girl.
```

File: wiki/Game_mode/en.md [hunk 5]
```diff
@@ -1,42 +1,50 @@
...
 
 ![Gameplay of osu!](/wiki/shared/osu-gameplay.jpg "osu! gameplay")
 
 ## ![][osu!taiko] osu!taiko
 
 *Main page: [osu!taiko](osu!taiko)*
 
-*osu!taiko* is based on the *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* game series produced by [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). Made to mimic the feel of playing a real drum, the player makes use of a keyboard or drum kit to hit differently-colored circles on a single conveyor.
+*osu!taiko* simulates the process of playing a real drum, and players make use of a keyboard or drum kit to hit differently-colored circles on a single conveyor. It is based on the *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* game series produced by [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). osu!taiko's mascot is [Mocha](/wiki/Mascots#-mocha), a green-haired girl in a kimono holding drumsticks.
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:31]:
```suggestion
*osu!taiko* simulates the process of playing a real drum, where players make use of a keyboard or drum kit to hit differently-coloured circles on a single conveyor. It is based on the *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* game series produced by [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). osu!taiko's mascot is [Mocha](/wiki/Mascots#-mocha), a green-haired girl in a kimono holding drumsticks.
```

File: wiki/Game_mode/en.md [hunk 6]
```diff
@@ -1,42 +1,50 @@
...
 
 ![Gameplay of osu!taiko](/wiki/shared/taiko-gameplay.jpg "osu!taiko gameplay")
 
 ## ![][osu!catch] osu!catch
 
 *Main page: [osu!catch](osu!catch)*
 
-*osu!catch* is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. The game mode consists of falling fruit of differing varieties and a "catcher" holding a plate to catch the fruit.
+In *osu!catch*, fruits of differing varieties fall to the beat from the top of the screen. The player controls a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. osu!catch's mascot is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:39]:
```suggestion
In *osu!catch*, fruits of differing varieties fall from the top of the screen, and the player must catch them to the beat. To this end they control a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching said fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. osu!catch's mascot is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```

File: wiki/Game_mode/en.md [hunk 7]
```diff
@@ -1,42 +1,50 @@
...
 
 ![Gameplay of osu!catch](/wiki/shared/catch-gameplay.jpg "osu!catch gameplay")
 
 ## ![][osu!mania] osu!mania
 
 *Main page: [osu!mania](osu!mania)*
 
-*osu!mania* is based on a variety of conveyor-type rhythm games (namely [Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia") and Beatmania). The game mode was chiefly introduced and implemented by [woc2006](https://osu.ppy.sh/users/1105845), and is made to mimic the feel of a real piano through the use of keyboard keys.
+*osu!mania* is a vertical scrolling rhythm game (VSRG) mode designed as a piano simulator. It was chiefly [introduced and implemented](https://osu.ppy.sh/community/forums/topics/100751) by [woc2006](https://osu.ppy.sh/users/1105845), and is based on a variety of conveyor-type rhythm games (namely [Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia") and Beatmania). osu!mania's mascots are twins [Mani and Mari](/wiki/Mascots#-mani-&-mari) with light purple to purple hair.
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:47]:
```suggestion
*osu!mania* is a vertical scrolling rhythm game (VSRG) mode designed as a piano simulator. It was chiefly [introduced and implemented](https://osu.ppy.sh/community/forums/topics/100751) by [woc2006](https://osu.ppy.sh/users/1105845), and is based on a variety of conveyor-type rhythm games (namely [Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia") and [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")). osu!mania's mascots are the twins [Mani and Mari](/wiki/Mascots#-mani-&-mari), both with purple and light purple hair.
```

File: wiki/Game_mode/en.md (hunk 4 above)
Comment (TicClick) [reply wiki/Game_mode/en.md]: *squints*

File: wiki/Game_mode/en.md (hunk 3 above)
Comment (TicClick) [reply wiki/Game_mode/en.md:17]: I guess

File: wiki/Game_mode/en.md (hunk 6 above)
Comment (TicClick) [reply wiki/Game_mode/en.md]: uhh kind of wanted to reduce the amount of `ca(t-)ching`s here, any other suggestion? I'm fine with the rest of the suggestion

**Review by TicClick — COMMENTED** [review]
yeah that's the stuff

File: wiki/Game_mode/en.md [hunk 8]
```diff
@@ -1,42 +1,50 @@
...
 
 ![Gameplay of osu!taiko](/wiki/shared/taiko-gameplay.jpg "osu!taiko gameplay")
 
 ## ![][osu!catch] osu!catch
 
 *Main page: [osu!catch](osu!catch)*
 
-*osu!catch* is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. The game mode consists of falling fruit of differing varieties and a "catcher" holding a plate to catch the fruit.
+In *osu!catch*, fruits of differing varieties fall from the top of the screen, and the player must catch them to the beat. To this end they control a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching said fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. osu!catch's mascot is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```
Comment (Walavouchey) [inline wiki/Game_mode/en.md:39]: > uhh kind of wanted to reduce the amount of `ca(t-)ching`s here, any other suggestion? I'm fine with the rest of the suggestion

better?

```suggestion
In *osu!catch*, fruits of differing varieties fall from the top of the screen, and the player must catch them to the beat. To this end they control a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching said fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. The mascot for this game mode is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```

Comment (TicClick) [reply wiki/Game_mode/en.md:39]: picked the first suggestion instead

**Review by Walavouchey — APPROVED** [review]

**Review by Joehuu — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**